examples/amp-link-rewriter.html

```diff
@@ -30,9 +30,21 @@
 
 </amp-link-rewriter>
 
-<p id="track-section">
-    <a class="content" href="https://www.amazon.de/dp/B01KU8Z8P8/">Stelton Theo Milk Jug</a>
-</p>
+<div id="track-section">
+    <h3>#track-section</h3>
+    <p>
+      <a class="content" href="https://www.amazon.de/dp/B01KU8Z8P8/">Amazon.de</a><br>
+      <a class="content" href="https://skip-example.com/test">This link is skipped by <code>href</code> attribute filter</a><br>
+      <a class="some-different-class" href="https://www.amazon.de/dp/B01KU8Z8P8/">This link is skipped by <code>class</code> filter</a>
+    </p>
+</div>
+<div id="do-not-track-section">
+  <h3>#do-not-track-section</h3>
+  <p>The links below do not belong to the section specified in <code>section</code> config param.</p>
+  <p>
+    <a class="content" href="https://amazon.com/dp/B01KU8Z8P8/">This link is skipped</a>
+  </p>
+</div>
```

Comment:
Is it necessary to change this example? If so, please make it more clear as to what should happen to each section based on this amp-link-rewriter config.

Reply:
I think I could just roll back this example.

Reply:
Done